installer/docker-installer.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+# authors:
+# Tony Vattathil tonynv@amazon.com
+# License: Apache 2.0
+
+set -e
+
+echo "[INITIALIZING taskcat installer!!]"
+
+INSTALL_DIR="/usr/local/bin"
+DOCKER_IMAGE_TAG="latest"
+
+if [ $# -eq 1 ]; then
+    if [ "${1}" = "--develop" ]; then
+        DOCKER_IMAGE_TAG="develop"
+        BINARY_SUFFIX="-develop"
+    else
+        echo "Usage: ${0} [--develop]"
+        exit 1
+    fi
+fi
+
+DOCKER_IMAGE=taskcat/taskcat:${DOCKER_IMAGE_TAG}
+
+if [ -d "${HOME}"/.aws ]; then
+    echo "[INFO] : Found boto profile)"
+    echo "[INFO] : Boto profile will map to container during execution"
+    echo "echo \"[dockerize]\"" > taskcat.docker
+    echo "docker run -it --rm -v ${HOME}/.aws:/root/.aws -v \$(pwd):/mnt ${DOCKER_IMAGE} taskcat \$@" >> taskcat.docker
+    echo "docker run -it --rm -v ${HOME}/.aws:/root/.aws -v \$(pwd):/mnt ${DOCKER_IMAGE} alchemist \$@" >> alchemist.docker
+else
+    echo "echo [dockerize]" > taskcat.docker
+    echo "echo [dockerize]" > alchemist.docker
+    echo "docker run -it --rm -v \$(pwd):/mnt ${DOCKER_IMAGE} taskcat \$@" >> taskcat.docker
+    echo "docker run -it --rm -v \$(pwd):/mnt ${DOCKER_IMAGE} alchemist \$@" >> alchemist.docker
+fi
+
+if [ "$(id -u)" -eq 0  ];then
+    docker pull ${DOCKER_IMAGE}
+    mv taskcat.docker ${INSTALL_DIR}/taskcat${BINARY_SUFFIX}
+    mv alchemist.docker ${INSTALL_DIR}/alchemist${BINARY_SUFFIX}
```

Comment:
I also removed the `sudo` on the above 2 lines because it's obviously not necessary if you are root.